doc/content/index.md

```diff
@@ -1,3 +1,16 @@
 !config navigation breadcrumbs=False scrollspy=False
 
-# PlatypusApp
+# Platypus class=center style=font-weight:200;font-size:400%
+
+Platypus enables the set-up and assembly of finite element problems using the
+[MFEM](https://github.com/mfem/mfem) finite element library from [MOOSE](https://github.com/idaholab/moose).
+
+!col! small=12 medium=12 large=12 icon=get_app
+
+## Getting Started class=center style=font-weight:200
```

Comment:
Is there a reason to use something other than the default from the CSS theme for these headlines?

Reply:
Since this page is the home page for visitors checking out the documentation, I used a similar style to the [Cardinal home page](https://github.com/neams-th-coe/cardinal/blob/devel/doc/content/index.md) for the base layout here, which uses this - the defaults are somewhat small for a homepage IMO, but I don't feel too strongly on this

Reply:
OK, that's fine.